pages/_app/index.jsx

```diff
@@ -178,18 +200,51 @@ class App extends NextApp {
     return null
   }
 
+  Login = () => {
+    const { router } = this.props
+
+    const search = new URLSearchParams({
+      reason: router.query.reason === 'logout' ? 'logout' : '',
+      loading: router.query.reason !== 'logout',
+      identity_provider_url: API_URL.replace('internal', ''),
```

Comment:
It's better to create a separate configuration property called `IDENTITY_PROVIDER_URL` and make the replacement there.

Reply:
I was thinking it and ended up with this approach. It's used only once. If we use this URL somewhere else we can create configuration property for it.